scripts/processing/06-make_epochs.py

```diff
@@ -150,6 +150,7 @@ def run_epochs(subject_id, tsss=False):
     print('  Getting rejection thresholds')
     reject = get_rejection_threshold(epochs.copy().crop(None, reject_tmax),
                                      random_state=random_state)
+    del reject['eog']  # Don't reject epochs based on EOG
```

Comment:
We probably still should reject based on EOG because we don't want to use trials with blinks. At least this was what we thought before.